orderbook/src/database/fees.rs

```diff
@@ -1,32 +1,46 @@
 use super::{orders::DbOrderKind, Postgres};
-use crate::{
-    conversions::*,
-    fee::{FeeData, MinFeeStoring},
-};
+use crate::{conversions::*, fee::{FeeData, MinFeeStoring, UnsubsidizedFee}};
 
-use anyhow::{anyhow, Context, Result};
-use bigdecimal::BigDecimal;
+use anyhow::{Context, Result};
 use chrono::{DateTime, Utc};
-use ethcontract::U256;
 use shared::maintenance::Maintaining;
 
+#[derive(sqlx::FromRow)]
+struct FeeRow {
+    pub gas_amount: f64,
+    pub gas_price: f64,
+    pub sell_token_price: f64,
```

Comment:
Note that these don't need to be `pub`. Fields are visible in the module (and sub-modules) already and the `FeeRow` type isn't public so can't be used outside anyway.

```suggestion
    gas_amount: f64,
    gas_price: f64,
    sell_token_price: f64,
```